index.html

```diff
@@ -379,6 +379,32 @@ <h2 id="deviceId">
           duplicate any existing values.
         </p>
       </section>
+      <section>
+        <h2 id="fullscreen">
+          Fullscreen Rendering considerations
+        </h2>
+          <p><i>This section is non-normative.</i></p>
+          <p>
+            For applications that support full screen rendering such as powerpoint applications,
+            the decision of the display surface that gets captured while in fullscreen mode is 
+            specific to the user agent implementation. A user agent MAY choose to switch from 
+            capture of a window to the fullscreen when the associated application goes 
+            fullscreen. For consistency reasons, an user agent must ensure that the display surface 
```

Comment:
Why be ambiguous as to what happens? I would imagine sharing a window that goes into fullscreen would be the same as if that window got maximized from the JS application's perspective.

My concern with "switching mode" is if this accidentally has other implications due to any differences in modes, if any other part of the spec says that anything should be done differently depending on mode.

If we switch to "fullscreen mode" maybe we accidentally leak information about other windows that shouldn't be shared, like OS UI or other windows that might cover the presentation. Or maybe this wording implicitly changes what audio to share, if we decide to support it.

Reply:
> I would imagine sharing a window that goes into fullscreen would be the same as if that window got maximized from the JS application's perspective.

It isn't from the perspective of the platform (at least in some cases).  The fullscreen view can be an entirely new window, rather than a resizing of the shared one.  Getting that right is tricky.